package.json

```diff
@@ -22,7 +19,6 @@
     "typescript": "~5.3.3"
   },
   "dependencies": {
-    "aws-cdk-lib": "2.121.1",
```

Comment:
Shouldn't we have it for infra to work?🤔

Reply:
well, we could, but this root package.json is not adding any dependencies to the functions, so I thought it was ok

Reply:
It's just that we will need it in infra🙂 Since we don't have a package.json in infrastructure folder right now, let's keep this package here